bin/setup

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
```

Comment:
I'm not sure this script lines up with the way we set up our databases in dev. See https://github.com/Safecast/safecastapi/wiki/Dev:-Setup-on-Linux-and-OS-X-using-Docker

Reply:
If someone wants to go native then they can use it here but again, this is a default file that comes with Rails 5, and we can delete it

Reply:
I think the way we initialize databases might be different enough to cause problems, so we should delete this.

https://github.com/Safecast/safecastapi/blob/master/bin/initialize_databases.bash